public/stylesheet.css

```
@@ -264,16 +264,27 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 55px;
+  width: 55px;
+  background-color: var(--color-background);
+  fill: var(--color-button-active);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-icon:focus {
+  cursor: pointer;
+  fill: var(--color-background);
+  background-color: var(--color-button-active);
+  border-radius: 50%;
+}
+
+.header__list-icon {
+  padding: 10px;
+  height: 55px;
+  width: 55px;
```

Is there any particular reason you repeated the exact pixel values here? Why not just do

```suggestion
  height: 100%;
  width: 100%;
```

---

no particular reason. I was tweaking the values and it stayed like that.

---

Just suggested it because now you can't mess up by changing the pixel value only in one place.

---

it's fine, I applied the suggestion. It's better not to have duplicates.